src/hooks/search/use-search.ts

```diff
@@ -1,6 +1,24 @@
 import { useState, useCallback } from 'react'
 import { type SearchDialogResult, type SearchApiResponse } from './types'
 
+function getItemUrl(item: SearchDialogResult): string {
```

Comment:
I liked the idea to put all together in one place but try to use `const` in utilitary methods to follow the project's pattern.
Example: `src/lib/utils.ts`